runtime/src/main/resources/banner.txt

```diff
@@ -1,8 +1,7 @@
-  ___                     ____                  _                  ____ _                 _
- / _ \ _ __   ___ _ __   / ___|  ___ _ ____   _(_) ___ ___  ___   / ___| | ___  _   _  __| |
-| | | | '_ \ / _ \ '_ \  \___ \ / _ \ '__\ \ / / |/ __/ _ \/ __| | |   | |/ _ \| | | |/ _` |
-| |_| | |_) |  __/ | | |  ___) |  __/ |   \ V /| | (_|  __/\__ \ | |___| | (_) | |_| | (_| |
- \___/| .__/ \___|_| |_| |____/ \___|_|    \_/ |_|\___\___||___/  \____|_|\___/ \__,_|\__,_|
-      |_|
+   _  __    ____   ____ _   ____    _____  ___
```

Comment:
is there a space here between "x" and "p"?

Reply:
Not a space there, actually It was generated automatically by tools. 
I can fix it manually to make it seems better.